readthedocs/notifications/messages.py

```diff
@@ -314,9 +302,9 @@ def get_rendered_body(self):
         body=_(
             textwrap.dedent(
                 """
-            The configuration key "build.image" is deprecated.
-            Use "build.os" instead to continue building your project.
-            Read more at https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os
+            The configuration key <code>build.image</code> is deprecated.
+            Use <code>build.os</code> instead to continue building your project.
+            Read more in our <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os">confg file documentation</a>.
```

Comment:
Typo and for consistency, `config` -> `configuration`

```suggestion
            Read more in our <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os">configuration file documentation</a>.
```